src/zc/table/column.py

```diff
@@ -150,7 +162,7 @@ def __init__(self, title=None, prefix=None, field=None,
         self.widget_extra = widget_extra
 
     def makeId(self, item):
-        return ''.join(self.idgetter(item).encode('base64').split())
+        return b64encode(self.idgetter(item).encode('utf-8')).decode('ascii')
```

Comment:
I have the feeling this is wrong.
I believe this should be a native string (str both in Python2 and Python3)

Reply:
Unicode seems fine to me?  Where do you see it causing problems on Python 2?

Reply:
base64 contains only ascii characters anyway, so this really shouldn't be a problem here.
The only difference is, that with the new implementation, the idgetter has to return unicorn if its non-ascii.
I'd propose to stick with the change @wosc as proposed by @wosc.